src/Examine.Core/ExamineManager.cs

```diff
@@ -110,7 +135,15 @@ public virtual void Stop(bool immediate)
                 {
                     // we don't want to kill the app or anything, even though it is terminating, best to just ensure that 
                     // no strange lucene background thread stuff causes issues here.
-                }                
+                }
+                try
+                {
+                    foreach (var suggester in RegisteredSuggesters.OfType<IDisposable>())
+                    {
+                        suggester.Dispose();
+                    }
+                }
+                catch { }
```

Comment:
If we don't need to log or throw the error here then there should be a comment clarifying that 😄 (Like the one above)